docs/rackhd/install_os.rst

```diff
@@ -150,3 +151,56 @@ Parameters  Type     Flags        Description
 switchName  String   **required** The name of the vswitch
 uplinks     String   *optional*   The array of vmnic# devices to set as the uplinks.(Ex: uplinks: ["vmnic0", "vmnic1"]). If an uplink is attached to a vSwitch, it will be removed from the old vSwitch before being added to the vSwitch named by 'switchName'.
 =========== ======== ============ ============================================
+
+Windows installation specifics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Mounting the OS Image**:
+Windows' installation requires that Windows OS' ISO image must be mounted to a directory accessable to the node.
+In the example below a windows server 2012 ISO image is being mounted to a directory name **Licensedwin2012**
+
+.. code-block:: REST
+
+    sudo mount -o loop /var/renasar/on-http/static/http/W2K2012_2015-06-08_1040.iso /var/renasar/on-http/static/http/Licensedwin2012
+
+* **Export the directory**
+Edit the samba config file in order to export the shared directory
+
+.. code-block:: REST
+
+sudo nano /etc/samba/smb.conf
+
+.. code-block:: REST
+
+    [windowsServer2012]
+        comment = not windows server 201
+        path = /var/renasar/on-http/static/http/Licensedwin2012
+        browseable = yes
+        guest ok = yes
+        writable = no
+        printable = no
+
+
+* **Restart the samba share**
+
+.. code-block:: REST
+
+    sudo service samba restart
+
+For **windows only** in payload:
```

Comment:
How about make a  windows payload section like 'Windows OS Installation Workflow Payload' , let it be in same level with  'Non-Windows OS Installation Workflow Payload' 
and for how to setup repo, we could have a independent section, we will also could add other examples to setup SLES,ESXi,RHEL repos.